### PR TITLE
[teleop] add reverse toggle to RA gui

### PR DESCRIPTION
**Pull request by emilyyuan03**
tested with lcm echo and got expected values
Screen Shot 2022-01-12 at 8 21 28 PM

### CODE REVIEW

**Review by CameronTressler — APPROVED** [review]

**Review by jnnanni — APPROVED** [review]

File: base_station/gui/src/components/ReverseToggle.vue [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  methods: {
+    setPart: function(enabled) {
+      this.$parent.publish("/reverse_drive", {
+        'type': 'ReverseDrive',
+        'reverse': enabled
+      })
+    }
+  }
+}
+</script>
```
Comment (cgiger00) [inline base_station/gui/src/components/ReverseToggle.vue:109]: would be nice if you could add screenshots of how this looks to your gui PRs

File: jetson/teleop/src/__main__.py [hunk 2]
```diff
@@ -129,6 +130,11 @@ def drive_control_callback(channel, msg):
 
         joystick_math(new_motor, magnitude, theta)
 
+        if(reverse):
```
Comment (cgiger00) [inline jetson/teleop/src/__main__.py:133]: why is there a reverse check in kill()?

Comment (CameronTressler) [reply jetson/teleop/src/__main__.py:133]: I don't think there is? Github may just be hiding it but this is in drive_control_callback

**Comment by CameronTressler** [comment]
Change going to be made first in ish-integration. From there it will be merged into main

**Pull request closed without merging**